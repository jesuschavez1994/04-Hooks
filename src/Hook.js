import React from 'react';

const Hook = () => {

 
  
    
    return (
        <>
            <h1>Hook</h1>
        </>
    )
}


export default Hook;