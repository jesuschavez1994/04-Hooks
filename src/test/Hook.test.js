import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Hook from '../Hook';


describe('Pruebas en <Hook />', () => {

    let wrapper = shallow(<Hook />);

    test('debe de mostrar correctamente ', () => {
        expect( wrapper ).toMatchSnapshot();
    });
    
})
