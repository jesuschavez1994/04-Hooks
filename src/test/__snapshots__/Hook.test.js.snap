// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Hook /> debe de mostrar correctamente  1`] = `
<Fragment>
  <h1>
    Hook
  </h1>
</Fragment>
`;
